fe/src/main/java/org/apache/doris/analysis/InPredicate.java

```diff
@@ -54,6 +54,8 @@
     private static final String IN = "in";
     private static final String NOT_IN = "not_in";
 
+    private static final NullLiteral NULL_LITERAL = new NullLiteral();
```

Comment:
Why use static variable?  I think local variable is enough.

Reply:
This object does not need to have one for every `InPredicate` instance.